packages/commonwealth/client/scripts/views/pages/notifications/notification_row_components.tsx

```diff
@@ -181,7 +181,8 @@ export const DefaultNotificationRow = (props: ExtendedNotificationRowProps) => {
           users={authorInfo.map(
             (auth) =>
               new AddressInfo({
-                id: null,
+                userId: 0, // TODO: is this OK?
```

Comment:
did we verify this?

Reply:
seems like we don't need userId in notifications, just the profile name